### PR TITLE
SDK fixes according to latest changes

### PR DESCRIPTION
**Pull request by kiseln**

### CODE REVIEW

File: bridge-sdk/connectors/omni-connector/src/omni_connector.rs [hunk 1]
```diff
@@ -179,6 +177,43 @@ impl OmniConnector {
...
+            .map_err(|_| BridgeSdkError::UnknownError)?;
+
+        let tx_id = near_rpc_client::change(
+            near_endpoint,
+            self.near_signer()?,
+            token_locker_id.to_string(),
+            "deploy_token".to_string(),
+            serialized_args,
+            300_000_000_000_000,
+            4_000_000_000_000_000_000_000_000,
```
Comment (frolvanya) [inline bridge-sdk/connectors/omni-connector/src/omni_connector.rs:205]: Do we really need that much attached deposit and has here?

Here's an example of `deploy_token` tx:
image

We only used 60 TGas and 4 NEAR looks like a big overkill. I believe that 1-2 NEAR should be enough to cover a storage cost

File: bridge-sdk/connectors/omni-connector/src/omni_connector.rs [hunk 2]
```diff
@@ -325,7 +365,7 @@ impl OmniConnector {
         }
 
         // TODO: Provide fee and nativeFee
-        let withdraw_call = factory.init_transfer(near_token_id, amount, 0, 0, receiver);
+        let withdraw_call = factory.init_transfer(erc20_address, amount, 0, 0, receiver, String::new());
```
Comment (frolvanya) [inline bridge-sdk/connectors/omni-connector/src/omni_connector.rs:368]: Let's add a proper fee in this PR. It should take too much time. What do you think?

File: bridge-sdk/connectors/omni-connector/src/omni_connector.rs [hunk 3]
```diff
@@ -376,7 +416,10 @@ impl OmniConnector {
             self.token_locker_id()?.to_string(),
             "sign_transfer".to_string(),
             serde_json::json!({
-                "nonce": origin_nonce.to_string(),
+                "transfer_id": {
+                    "origin_chain": "Near", // TODO: provide transfer_id instead of only nonce
```
Comment (frolvanya) [inline bridge-sdk/connectors/omni-connector/src/omni_connector.rs:420]: Providing a chain by string feels very insecure. Can we use `ChainKind` enum here?

File: bridge-sdk/connectors/omni-connector/src/omni_connector.rs (hunk 1 above)
Comment (kiseln) [reply bridge-sdk/connectors/omni-connector/src/omni_connector.rs:205]: I believe new token account requires 3 NEAR deposit

Comment (frolvanya) [reply bridge-sdk/connectors/omni-connector/src/omni_connector.rs:205]: Then why we attach 4 here and why 300 TGas?

Comment (kiseln) [reply bridge-sdk/connectors/omni-connector/src/omni_connector.rs:205]: @karim-en how much would you recommend to attach for deploying the token? I think it's slightly more than 3 NEAR for the token account

File: bridge-sdk/connectors/omni-connector/src/omni_connector.rs (hunk 2 above)
Comment (kiseln) [reply bridge-sdk/connectors/omni-connector/src/omni_connector.rs]: Done

File: bridge-sdk/connectors/omni-connector/src/omni_connector.rs (hunk 3 above)
Comment (kiseln) [reply bridge-sdk/connectors/omni-connector/src/omni_connector.rs]: Done

**Review by frolvanya — APPROVED** [review]

File: bridge-sdk/connectors/omni-connector/src/omni_connector.rs (hunk 1 above)
Comment (frolvanya) [reply bridge-sdk/connectors/omni-connector/src/omni_connector.rs:205]: It looks like a hard const to me, so I thought that we can always attach exactly 3 NEAR. Am I wrong? If it's a bit more than 3, then you're correct and there's no point in attaching less and calculating these decimals. We can leave as it is 4 NEARs and everything else will be refunded in any case

image

Comment (karim-en) [reply bridge-sdk/connectors/omni-connector/src/omni_connector.rs:205]: For now let's hardcode it as 4.
I'm going to improve it by reducing this amount and add a refund.

**Pull request merged**